src/index.js

```diff
@@ -60,7 +60,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(/\\/g, '\\\\')}';\n`;
```

Comment:
The regex replace works great on my windows box. The `JSON.stringify` doesn't - didn't try to dig into why (though it's a cool trick).